File: Benchmarks/enterprise/java/spring/testcode/BenchmarkTest59199.java
// SPDX-License-Identifier: Apache-2.0
import jakarta.servlet.http.HttpServletRequest;
import jakarta.servlet.http.HttpServletResponse;
import org.springframework.web.bind.annotation.*;

@RestController
public class BenchmarkTest59199 {
    private static class GraphQLRequest {
        public String query;
        public java.util.Map<String, Object> variables;
        public GraphQLRequest() {}
    }

    @PostMapping(path="/BenchmarkTest59199", consumes="application/json")
    public void BenchmarkTest59199(@RequestBody GraphQLRequest req, HttpServletRequest request, HttpServletResponse response) throws Exception {
        response.sendError(500, "Internal error");
    }
}

File: Benchmarks/enterprise/python/fastapi/testcode/benchmark_test_59199.py
# SPDX-License-Identifier: Apache-2.0
from fastapi import Request
from pydantic import BaseModel


class UserInput(BaseModel):
    payload: str = ''

async def BenchmarkTest59199(request: Request, req: UserInput):
    json_value = req.payload
    eval(str(json_value))
    return {"updated": True}
